librocksdb_sys/crocksdb/c.cc

```diff
@@ -5853,4 +5857,42 @@ void ctitandb_delete_files_in_ranges_cf(
   SaveError(errptr, static_cast<TitanDB*>(db->rep)->DeleteFilesInRanges(
                         cf->rep, &ranges[0], num_ranges, include_end));
 }
+
+/* RocksDB Cloud */
+#ifdef USE_CLOUD
+struct crocksdb_cloud_envoptions_t {
+  CloudEnvOptions rep;
+};
+
+crocksdb_env_t* crocksdb_cloud_aws_env_create(
+    crocksdb_env_t* base_env, crocksdb_cloud_envoptions_t* cloud_options,
+    char** errptr) {
+  // Store a reference to a cloud env. A new cloud env object should be
+  // associated with every new cloud-db.
+  std::unique_ptr<CloudEnv> cloud_env;
+
+  CloudEnv* cenv;
+  if (SaveError(errptr, CloudEnv::NewAwsEnv(base_env->rep, cloud_options->rep,
```

Comment:
the list of params are much different to in the cloud code https://github.com/tikv/rocksdb/blob/6.4.cloud/include/rocksdb/cloud/cloud_env_options.h#L291 does the code really being built?